app/views/admin/action_items/_rejected_auctions.html.erb

```diff
@@ -15,7 +15,7 @@
       <td><%= link_to(auction.title, admin_auction_path(auction.id)) %></td>
       <td><%= auction.delivery_due_at %></td>
       <td><%= auction.delivery_url %></td>
-      <td><%= link_to auction.winning_bidder_name, admin_user_path(auction.winning_bidder_id) %></td>
+      <td><% if auction.winning_bidder? %><%= link_to auction.winning_bidder_name, admin_user_path(auction.winning_bidder_id) %><% end %></td>
```

Comment:
this may require using a different method name / presenters (eg: you call `winning_bidder.link` and `UserPresenter` returns a link and `GuestPresenter` returns a string)